eos/effectHandlerHelpers.py

```diff
@@ -160,13 +159,6 @@ def remove(self, mod):
         for i in xrange(oldPos, len(self)):
             self[i].position -= 1
 
-    def toDummy(self, index):
```

Comment:
Why was this moved to the Module class? Methinks this should remain as a property of the collection.